lib/modules/apostrophe-images/lib/cursor.js

```diff
@@ -40,6 +40,65 @@ module.exports = {
       }
     });
 
+    self.addFilter('orientation', {
+      finalize: function() {
+        const orientation = self.get('orientation');
+        if (!orientation) {
+          return;
+        }
+
+        self.and(getCriteria(orientation));
+
+        function getCriteria(orient) {
```

Comment:
orient should be orientation

Reply:
That was to avoid same name for param and higher level constant but ok.